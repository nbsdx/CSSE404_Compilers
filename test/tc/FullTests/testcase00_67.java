class Test {
    public static void main(String[] args) {
	int x = 0;
	//args is just a placeholder... should not be able to be accessed
	System.out.println(args);
    }
}
