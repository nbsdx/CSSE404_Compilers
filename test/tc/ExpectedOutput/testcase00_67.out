Variable args is not declared.
In MiniJava, System.out.println only takes an int.  The expression has type null
